### ISSUE TITLE
[MAS2.4.3] [Focus Order - Sign in with Azure account dialog] Focus order is not logical after clicking on the "Sign In with your Azure account".

### ISSUE THREAD

**Issue by ShikhaMishra11** [microsoft/BotFramework-Emulator#1953]
**Actual Result:**
After clicking on the "Sign In with your Azure account", the focus stays on the same control and reaches into the pop-up/dialog after navigating the whole control of the "Welcome" tab.
As well as the focus comes out of the pop-up/dialog with tab key without closing the pop-up/dialog.

**Expected Result:**
As user activates the "Sign In with your Azure account" button the focus should lands on the first interactive element of the pop-up. 
As focus reaches the last element it shouldn't goes out of the pop-up. In case the focus goes out of the pane the pop-up should gets closed.

**Note:**
Same issue observed with pop-up and dialog's throughout the application.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to the "Sign In with your Azure account" and activate it.
4. Navigate with keyboard.
5. Observe the issue.

**User Impact:**
User who rely on keyboard would be impacted as focus order is not in the ideal sequential order.

**Test Environment:**
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.5.2-linux

**Impacted MAS Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame2.aspx?sourcedoc=%7B0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9%7D](url)

**Attachment:**
[Focus order is not logical.zip](https://github.com/microsoft/BotFramework-Emulator/files/3794128/Focus.order.is.not.logical.zip)

**Comment by tonyanziano** [comment]
This has been fixed already and should be verified in the latest version.

**Video:**
[1953-fix.zip](https://github.com/microsoft/BotFramework-Emulator/files/3934452/1953-fix.zip)

**Comment by ShikhaMishra11** [comment]
We have checked in the latest nightly, the issue is fixed now. So, we are closing the bug.

Test Environment:
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.6.0.94518
